File: app/src/test/java/com/github/colinjeremie/willyoufindit/activities/TutoActivityTest.java
package com.github.colinjeremie.willyoufindit.activities;

import android.os.Build;
import android.support.v4.view.ViewPager;

import com.github.colinjeremie.willyoufindit.BuildConfig;
import com.github.colinjeremie.willyoufindit.R;
import com.github.colinjeremie.willyoufindit.adapters.TutoAdapter;

import junit.framework.Assert;

import org.junit.Before;
import org.junit.Test;
import org.junit.runner.RunWith;
import org.robolectric.Robolectric;
import org.robolectric.RobolectricGradleTestRunner;
import org.robolectric.annotation.Config;

@RunWith(RobolectricGradleTestRunner.class)
@Config(constants = BuildConfig.class, sdk = Build.VERSION_CODES.LOLLIPOP)
public class TutoActivityTest {

    private ViewPager mViewPager;

    @Before
    public void setUp(){
        TutoActivity activity = Robolectric.setupActivity(TutoActivity.class);
        mViewPager = (ViewPager) activity.findViewById(R.id.view_pager);
    }

    @Test
    public void shouldHave3Pages() throws Exception {
        Assert.assertEquals(mViewPager.getAdapter().getCount(), TutoAdapter.COUNT);
    }
}

File: app/src/test/java/com/github/colinjeremie/willyoufindit/activities/TutoActivityTest.kt
package com.github.colinjeremie.willyoufindit.activities

import android.os.Build
import android.support.v4.view.ViewPager

import com.github.colinjeremie.willyoufindit.BuildConfig
import com.github.colinjeremie.willyoufindit.R
import com.github.colinjeremie.willyoufindit.adapters.TutoAdapter

import junit.framework.Assert

import org.junit.Before
import org.junit.Test
import org.junit.runner.RunWith
import org.robolectric.Robolectric
import org.robolectric.RobolectricTestRunner
import org.robolectric.annotation.Config

@RunWith(RobolectricTestRunner::class)
@Config(constants = BuildConfig::class, sdk = intArrayOf(Build.VERSION_CODES.LOLLIPOP))
class TutoActivityTest {

    private lateinit var mViewPager: ViewPager

    @Before
    fun setUp() {
        val activity = Robolectric.setupActivity(TutoActivity::class.java)
        mViewPager = activity.findViewById(R.id.view_pager)
    }

    @Test
    @Throws(Exception::class)
    fun shouldHave3Pages() {
        Assert.assertEquals(mViewPager.adapter!!.count, TutoAdapter.COUNT)
    }
}